core/modules/path/path.admin.inc

```diff
@@ -348,13 +348,13 @@ function path_patterns_form($form, $form_state) {
   $config = config('path.settings');
 
   $form['help'] = array(
-    '#markup' => '<p>' . t('URL patterns automatically alias new content based on wildcards called "tokens". For example the URL "node/10" might be automatically aliased to "blog/my-first-post" using the pattern "blog/[node:title]".') . '</p>',
+    '#markup' => '<p>' . t('Your site can have URLs generated automatically based on wildcards called <em>tokens</em> or <em>URL aliases</em> can be defined manually in the UI. For automatic generation of URL aliases, define <em>URL alias patterns</em> for each type of content below.') . '</p>',
```

Comment:
Personally, I find the new text here more confusing. The verbosity isn't helping and muddles the concept more than it helps. The previous PR only updated the "alias" terminology, could we stick to that?